python_modules/libraries/dagster-dg/docs/util/types.ts

```diff
@@ -0,0 +1,15 @@
+export type Contents = Package[];
+
+export type Package = {
+  name: string;
+  componentTypes: ComponentType[];
+};
+
+export type ComponentType = {
+  name: string;
+  author: string;
+  example: string;
+  schema: string;
+  description: string;
+  tags: string[];
+};
```

Comment:
It would be sick if these types could be generated from a backend type to make sure the python / TS boundary here is always properly typed.